src/app/commands/cmd_import_sprite_sheet.cpp

```diff
@@ -144,14 +152,36 @@ class ImportSpriteSheetWindow : public app::gen::ImportSpriteSheet
       std::max<int>(1, h));
   }
 
+  gfx::Size getPaddingFromEntries() {
+  int padW = horizontalPadding()->textInt();
+  int padH = verticalPadding()->textInt();
+    if (padW < 0) {
+      horizontalPadding()->setText("0");
+      padW = 0;
+    }
+    if (padH < 0) {
+      verticalPadding()->setText("0");
+      padH = 0;
+    }
+    return gfx::Size(padW, padH);
+  }
+
   void onEntriesChange() {
     m_rect = getRectFromEntries();
+    m_padding = getPaddingFromEntries();
 
     // Redraw new rulers position
     if (m_editor) {
       EditorStatePtr state = m_editor->getState();
       if (SelectBoxState* boxState = dynamic_cast<SelectBoxState*>(state.get())) {
         boxState->setBoxBounds(m_rect);
+        if (int(boxState->getFlags()) & int(SelectBoxState::Flags::PaddingRulers)) {
```

Comment:
I think this condition is always true, because when we are in the `ImportSpriteSheetWindow`, if `m_editor != nullptr` it means that we have changed the state of `m_editor` to `SelectBoxState` (from `ImportSpriteSheetWindow::captureEditor()`) with this `PaddingRulers` flag enabled. A possible nice change would be to have an assert:
```
ASSERT(int(boxState->getFlags()) & int(SelectBoxState::Flags::PaddingRulers));
```
Anyway as the next line (`boxState->setPaddingBounds()`) already does this, we can just remove this line/if-condition.

Another note: About the code that was already there, I'm not sure why I put a check `if (SelectBoxState* boxState = dynamic_cast<SelectBoxState*>(state.get()))`, it's like the state could be different to `SelectBoxState` in some special cases when `m_editor != nullptr`, but I don't remember why I put that check.